README.md

```diff
@@ -65,7 +65,7 @@ Configure the checks for libraries in the [Arduino Library Manager](https://gith
 #### Supported values
 
 - `submit` - Also run additional checks required to pass before a library is accepted for inclusion in the index.
-- `update`- Also run additional checks required to pass before a library is accepted for inclusion in the index.
+- `update`- Also run additional checks required to pass before an update to a library is accepted.
```

Comment:
Thanks! That was a copy/paste error on my part.
```suggestion
- `update`- Also run additional rules required to pass before new releases of a library already in the index are accepted.
```
I think this explanation from the [Arduino Lint command line help](https://arduino.github.io/arduino-lint/latest/commands/arduino-lint/#options) is a little bit more clear.

Reply:
Agreed, I was trying to abbreviate https://arduino.github.io/arduino-lint/latest/#library-manager-setting